Sixteen750/src/main/java/org/firstinspires/ftc/sixteen750/Hardware.java

```diff
@@ -15,6 +15,7 @@
 import java.util.List;
 import org.firstinspires.ftc.robotcore.external.navigation.VoltageUnit;
 
+
```

Comment:
Don't just add whitespace. This is annoying in history. I'd suggest you run "bun format" before you commit, so that everything just gets autoformatted :)